recipes/olc-pge/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conans import ConanFile, tools, errors
+import os
+
+required_conan_version = ">=1.32.0"
+
+
+class olcPixelGameEngineConan(ConanFile):
+    name = "olc-pge"
+    homepage = "https://github.com/OneLoneCoder/olcPixelGameEngine"
+    description = "The olcPixelGameEngine is a single-file prototyping and game-engine framework created in C++."
+    topics = ("conan", "olc", "pge", "pixelengine", "pixelgameengine", "pgex",
+              "game-development", "game-engine", "engine", "gamedev", "gaming", "graphics")
+    license = "LicenseRef-LICENSE"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "image_loader": ["png", "stb", "gdi"],
+    }
+    default_options = {
+        "image_loader": "png",
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        # Default option values.. (can be overridden by downstream)
+        if self.settings.os == "Windows":
+            self.options.image_loader = "gdi"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 14)
+        if self.settings.os == "Linux" and tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version < "18":
+            raise errors.ConanInvalidConfiguration(
+                "Requires a system with recent OpenGL.")
```

Comment:
@grafikrobot
Why are you excluding older ubuntu releases?
Is the opengl version too low?
Are there bugs in previous mesa releases?
What opengl version/extensions does it require that are missing from older releases?